audformat/core/utils.py

```diff
@@ -372,11 +373,12 @@ def index_has_overlap(
     if index_type(index) == define.IndexType.FILEWISE:
         return False
 
-    index = to_segmented_index(index, allow_nat=False)
+    index = to_segmented_index(index)
```

Comment:
Can be removed as we already know that it must be a segmented index.

Reply:
Yea already did, see latest commit

Reply:
What if the user passes a data frame that index ain't conform to audformat ? This keeps us in the safe side, or is it also a costly operation ?

Reply:
Not really, since `to_segmented_index()` also expects an index that is conform to audformat.

Reply:
Ok, applied.